fence/sync/sync_users.py

```diff
@@ -344,8 +464,9 @@ def _parse_yaml(self, filepath, encrypted=True):
         user_info = dict()
         user_policies = dict()
 
-        with self._read_file(filepath, encrypted=encrypted) as stream:
-            data = yaml.safe_load(stream)
+        dbgap_key = getattr(self, "dbgap_key")
```

Comment:
```suggestion
        dbgap_key = getattr(self, "dbgap_key", None)
```

another minor consistency thing to match above